ee/codegen_integration/fixtures/simple_code_execution_node/code/display/nodes/code_execution_node.py

```diff
@@ -1,22 +1,32 @@
 from uuid import UUID
 
 from vellum_ee.workflows.display.nodes import BaseCodeExecutionNodeDisplay
-from vellum_ee.workflows.display.nodes.types import PortDisplayOverrides
+from vellum_ee.workflows.display.nodes.types import NodeOutputDisplay, PortDisplayOverrides
 from vellum_ee.workflows.display.vellum import NodeDisplayData, NodeDisplayPosition
 
 from ...nodes.code_execution_node import CodeExecutionNode
 
 
 class CodeExecutionNodeDisplay(BaseCodeExecutionNodeDisplay[CodeExecutionNode]):
     code_input_id = UUID("19b05769-cee3-4659-80d1-66fcae4e27c3")
+    arg_input_id = UUID("78da07ee-cc77-445e-af85-f60ab4f7a59f")
```

Comment:
To go with my comment above, this should not be here and we should instead be generating a class attribute called something like `node_input_ids_by_name` similar to `BaseTemplatingNodeDisplay.input_ids_by_name`

Reply:
Updated